src/features/location/components/LocationWidget.tsx

```diff
@@ -23,9 +23,8 @@ export const LocationWidget: React.FC = () => {
   const [widgetWidth, setWidgetWidth] = React.useState<number | undefined>()
   const { isSplashScreenHidden } = useSplashScreenContext()
 
-  const { userPosition } = useLocation()
-  const isGeolocated = !!userPosition
-  const locationTitle = isGeolocated ? 'Ma position' : 'Me localiser'
+  const { getLocationTitle, isGeolocated, isCustomPosition } = useLocation()
+  const locationTitle = getLocationTitle({ is: 'Ma position', isnot: 'Me localiser' })
```

Comment:
La méthode `getLocationTitle` servira-t-elle ailleurs ? Peut-on la renommer `getLocationWidgetTitle` ?

Reply:
Il est assez large pour l'instant, donc il peut potentiellement être utilisé ailleurs. Ça serait getLocationWidgetTitle 
 si ça faisait : () => getLocationTitle({ is: 'Ma position', isnot: 'Me localiser' }), je pense

Reply:
Résolu car on l'a bougé en MOB dans le LocationWidget directement